errorStrings/errorStrings.json

```diff
@@ -0,0 +1,27 @@
+﻿{
+    "CommandExecutionFailed": "Command {0} failed with error code {1}",
```

Comment:
Does it make sense to have the strings in a separate file if we are not planning on supporting translations of the messages? If we put them inline it'll be easier to see that this one needs two parameters for example.